package.json

```diff
@@ -43,16 +43,16 @@
     "debug": "^4.3.4",
     "lodash": "^4.17.15",
     "mkdirp": "^1.0.4",
-    "rimraf": "^3.0.2",
+    "rimraf": "^5.0.5",
```

Comment:
Hi @aman19K this package updated to a MAJOR version. Have we validated the breaking changes of the respective package and our use case?

Reply:
Yes @antonyagustine , we're only using it for removing dist folder.